### PR TITLE
Create euler-6-proposal

### PR DESCRIPTION
**Pull request by serejandmyself**

### CODE REVIEW

File: site/content/post/euler-6-proposal.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+The migration of the precommits is complex. To avoid any additional changes in the distribution, we propose to save the state of the precommits in [a .csv file](https://github.com/cybercongress/launch-kit/tree/0.1.0/game_rewards_calculations) at a certain block height and use it for the Game of Links recalculations after the Game. Thus, we can avoid major changes in the distribution and make the migration a smoother process.
+
+The stake of the validators should be burned. The complexity of changing distribution parameters is much higher than the error of the final stake results.
+
+Additionally, we are now able to test the genesis ceremony (LINK - WHAT IS IT?) and parameters discussion. The [launch-kit repo](https://github.com/cybercongress/launch-kit) is ready for newcomers and it seems possible to do this now.
+
+The workflow should be fairly easy:
+1. Announce network halt at a given block height
+2. Export precommits and cyberlinks on a given block
+3. Import the UI for cyber.page to cyberlinks (NEEDS EXPLANATION)
```
Comment (SaveTheAles) [inline site/content/post/euler-6-proposal.md]: It means we need some form in the cyber.page for users relink process in e-6

File: site/content/post/euler-6-proposal.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+4. Create a guide for importing cyberlinks via the CLI (WE CREATE IT?)
```
Comment (SaveTheAles) [inline site/content/post/euler-6-proposal.md]: Will do

File: site/content/post/euler-6-proposal.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+4. Generate Genesis
+5. Genesis ceremony (LINK)
```
Comment (SaveTheAles) [inline site/content/post/euler-6-proposal.md]: https://github.com/cybercongress/launch-kit/blob/0.1.0/gen_txs/README.md

**Comment by serejandmyself** [comment]
Updated with comments from @SaveTheAles

File: site/content/post/euler-6-proposal.md (hunk 1 above)
Comment (Aha203) [reply site/content/post/euler-6-proposal.md]: It should be done by the original authors of cyberlinks.

Comment (SaveTheAles) [reply site/content/post/euler-6-proposal.md]: Supposed to, but this method has a weakness I described in the doc. So cyberlinks owners or pioneers should carry about migrating their links asap in the new network. It's in their interest.

File: site/content/post/euler-6-proposal.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+The migration of the precommits is complex. To avoid any additional changes in the distribution, we propose to save the state of the precommits in [a .csv file](https://github.com/cybercongress/launch-kit/tree/0.1.0/game_rewards_calculations) at a certain block height and use it for the Game of Links recalculations after the Game. Thus, we can avoid major changes in the distribution and make the migration a smoother process.
+
+The stake of the validators should be burned. The complexity of changing distribution parameters is much higher than the error of the final stake results.
+
+Additionally, we are now able to test the genesis ceremony (LINK - WHAT IS IT?) and parameters discussion. The [launch-kit repo](https://github.com/cybercongress/launch-kit) is ready for newcomers and it seems possible to do this now.
+
+The workflow should be fairly easy:
+1. Announce network halt at a given block height
+2. Export precommits and cyberlinks on a given block
```
Comment (Aha203) [inline site/content/post/euler-6-proposal.md]: do we need to export the stake of validators at this stage too?

Comment (SaveTheAles) [reply site/content/post/euler-6-proposal.md]: No, we will not do it. As I explained in the doc:

> The stake of the validators should be burned. The complexity of changing distribution parameters is much higher than the error of the final stake results.

This way the accounts part of e-6 genesis file will the same as e-5.

**Pull request merged**